apps/projects/app/components/Content/IssueDetail/EventsCard.js

```diff
@@ -133,10 +144,11 @@ const activities = (
                 </Text.Block>
               )}
               <Tag
+                uppercase={false}
                 color={`${theme.surfaceContentSecondary}`}
                 background={`${theme.border}`}
               >
-                Quality: {data.review.rating}
+                {'Quality:' + ' ' + workRatings[data.review.rating]}
```

Comment:
IIn this particular case text inside <Tag> was `Quality:, 5 - Excellent` - somehow an extra coma is always added. My guess (I've not investigated too much) is that number of children >1 produces coma-separated list instead of nicely concatenating...

Reply:
Ohhh, yeah, I forgot about that.